src/libstd/sys/unix/ext/process.rs

```diff
@@ -67,10 +67,20 @@ pub trait CommandExt {
     /// an error indicating why the exec (or another part of the setup of the
     /// `Command`) failed.
     ///
+    /// `exec` not returning has the same implications as calling
+    /// [`process::exit`] – no destructors on the current stack or any other
+    /// thread’s stack will be run. Therefore, it is recommended to only call
+    /// `exec` at a point where it is fine to not run any destructors. Note,
+    /// that `execvp` syscall independently guarantees that all memory is freed
+    /// and all file descriptors with the `CLOEXEC` option (set by default on all
+    /// file descriptors opened by the standard library) are closed.
```

Comment:
Is this platform specific behavior? I'm not familiar with how all the platforms handle `execvp`